docs/source/configuration/environmentvariables.md

```diff
@@ -171,21 +171,22 @@ As a result, your app will load the add-ons in the following order:
 - `@eeacms/volto-accordion-block`
 - `volto-slate`
 
-!!! important
-    `ADDONS` does *not* work for development packages, which are *always* enabled if defined in your `jsconfig.json` or via `mrs.developer.json`.
+```{important}
+`ADDONS` does *not* work for development packages, which are *always* enabled if defined in your `jsconfig.json` or via `mrs.developer.json`.
```

Comment:
I find this line misleading. ADDONS shouldn't care if an addon comes from node_modules or is aliased via jsconfig. 

I think the message is "The ADDONS key is a Volto specific configuration. Simply setting ADDONS doesn't download the javascript package. This has to be covered by another way, either installing the addon package (with `yarn add`) or loading it as a development package with mrs-developer.

Reply:
I wanted to state clearly that a developer add-on is always enabled, if declared in mrs-developer and thus, in jsconfig.js. I'll try to merge both ideas.

Reply:
but maybe that clarification has to be done elsewhere, not referred to ADDONS env var...